starters/express-apollo-prisma/src/graphql/mappers/technology.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { mapTechnology } from './technology';
+import { TechnologyEntity } from '@prisma/client';
+import { Technology } from '../schema/generated/types';
+
+describe('.mapTechnology', () => {
+	describe('when called', () => {
+		it('returns expected result', () => {
+			const MOCK_TECHNOLOGY: TechnologyEntity = {
+				id: 123,
+				name: 'MOCK_TECHNOLOGY',
```

Comment:
The mapper converts whatever is in the "name" field to a string. Should we add test cases to ensure that it works as expected?

Reply:
Then we might as well drop the string conversion from line 6

Reply:
You're too fast 😅  I tried to change my reply before you could read it. Totally read the code wrong.